07_hello_friend/hello_friend_spec.rb

```diff
@@ -96,6 +96,7 @@
   end
 
   it "says hello to someone" do
-    Friend.new.greeting("Bob").should == "Hello, Bob!"
+    "I am here"
```

Comment:
Huh ?

Reply:
This was a test just for me. I forgot to delete:((